7.2/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM circleci/php:7.2-apache
```

Comment:
Should we start using the new images? https://circleci.com/docs/2.0/circleci-images/
Or create a maintenance task to do it?

Reply:
I guess a new task for it will be fine. Thanks for your idea!